ibft/storage/store.go

```diff
@@ -155,85 +153,6 @@ func (i *ibftStorage) CleanAllInstances(msgID []byte) error {
 	return nil
 }
 
-// SaveLastChangeRoundMsg updates last change round message
-func (i *ibftStorage) SaveLastChangeRoundMsg(msg *specqbft.SignedMessage) error {
-	i.forkLock.RLock()
-	defer i.forkLock.RUnlock()
-
-	var signers [][]byte
-	for _, s := range msg.GetSigners() {
-		signers = append(signers, uInt64ToByteSlice(uint64(s)))
-	}
-	signedMsg, err := i.fork.EncodeSignedMsg(msg)
-	if err != nil {
-		return errors.Wrap(err, "could not encode signed message")
-	}
-	return i.save(signedMsg, lastChangeRoundKey, msg.Message.Identifier, signers...)
-}
-
-// GetLastChangeRoundMsg returns last known change round message
-func (i *ibftStorage) GetLastChangeRoundMsg(identifier []byte, signers ...spectypes.OperatorID) ([]*specqbft.SignedMessage, error) {
-	i.forkLock.RLock()
-	defer i.forkLock.RUnlock()
-
-	if len(signers) == 0 {
-		res, err := i.getAll(lastChangeRoundKey, identifier[:])
-
-		if err != nil {
-			return nil, err
-		}
-		return res, nil
-	}
-
-	var res []*specqbft.SignedMessage
-	for _, s := range signers {
-		msg, found, err := i.get(lastChangeRoundKey, identifier[:], uInt64ToByteSlice(uint64(s)))
-		if err != nil {
-			return res, err
-		}
-		if !found {
-			return res, nil
-		}
-		sm := new(specqbft.SignedMessage)
-		if err := sm.Decode(msg); err != nil {
-			return res, err
-		}
-		res = append(res, sm)
-	}
-	return res, nil
-}
-
-// CleanLastChangeRound cleans last change round message of some validator, should be called upon controller init
-func (i *ibftStorage) CleanLastChangeRound(identifier []byte) error {
```

Comment:
We don't need to remove this as it should be run in the migration. We should take all shares, go through them and run this function for each share's identifier.